packages/hierarchies/src/test/internal/operators/grouping/BaseClassGrouping.test.ts

```diff
@@ -105,7 +105,7 @@ describe("BaseClassGrouping", () => {
         }),
       ];
       const ecClass = { fullName: "TestSchema.ParentClass", label: "ParentClass" } as unknown as ECClass;
-      expect(await baseClassGrouping.createBaseClassGroupsForSingleBaseClass(nodes, ecClass, new BaseClassChecker(metadataProvider))).to.deep.eq({
+      expect(await baseClassGrouping.createBaseClassGroupsForSingleBaseClass(nodes, ecClass, new BaseClassChecker(metadataProvider, 1000))).to.deep.eq({
```

Comment:
Maybe change `BaseClassChecker` to default the size to `0`? Majority of tests don't care about the cache.

Reply:
Done

Reply:
Sorry, I wasn't clear. My suggestion was to make `cacheSize` argument in the constructor of `BaseClassChecker` option and default to 0. That way tests don't need to pass anything.

Reply:
Ok. Done